scraper.py

```diff
@@ -0,0 +1,255 @@
+            company = company.replace(" ", "").split("·")
+            if len(company) == 2:
+                experience_dict["company"] = company[0]
+                experience_dict["employmentType"] = company[1]
+            
+            if len(company) == 1:
+                if company[0] in ["Full-time", "Part-time", "Self-employed", "Freelance", "Internship", "Trainee"]:
+                    experience_dict["employmentType"] = company[0]
+                else:
+                    experience_dict["company"] = company[0]
+            
+            ### DURATION ###
+            duration = temp[2].text
+            duration = duration.replace(" ", "").split("·")
+            start_end = duration[0].split("-")
+            experience_dict["startDate"] = start_end[0]
+            experience_dict["endDate"] = start_end[1]
+            experience_dict["duration"] = duration[1]
+
+            ### LOCATION ###
+            location = temp[3].text
+            location = location.replace(" ", "").split("·")
+            
+            # if both locationType and location are present
+            if len(location)==2:
+                experience_dict["location"] = location[0]
+                experience_dict["locationType"] = location[1]
+            
+            ### taken care of the case where one of the locationType or location is not present
+            if len(location)==1: 
+                if location[0] in ["Remote", "On-site", "Hybrid"]:
+                    experience_dict["locationType"] = location[0]
+                else:
+                    experience_dict["location"] = location[0]   
+
+            experience_list.append(experience_dict)
+
+        return experience_list
+    
+    #### FUNCTIONS TO COMPLETE ####
+
+    def get_education(self):
+        pass
+
+    def get_skills(self):
+        pass
+
+    ### FUNCTION TO COMPLETE ###
+
+    def get_output_in_json(self, object):
+        from json import dumps
+
+        print("converting to json...")
+        if type == str:
+            return dumps(object, indent=4)
+        if type == "array":
+            for i in range(len(object)):
+                object[i] = dumps(object[i], indent=4)
+        print("converted to json...")
+        return dumps(object, indent=4)
+
+
+if __name__ == "__main__":
+    ### PARSING ARGUMENTS ##
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--running",
+        type=bool,
+        help="Take control of the already running chrome instance in debug mode with LinkedIn signed in",
+        default=False,
+    )
+    parser.add_argument(
+        "--portnumber",
+        type=int,
+        help="Port number of the already running chrome instance in debug mode with LinkedIn signed in",
+        default=9222,
+    )
+
+    args = parser.parse_args()
+
+    ### ARGUMENTS PARSED ###
+
+    ### GETTING DRIVERS ###
+
+    if not args.running:
+        config = ConfigParser()
+        config.read("config.ini")
```

Comment:
Create both `example.config.ini` and `config.ini`. 

And then only include `config.ini` in `.gitignore`.

Reply:
ok done